ocsf-import/package.yaml

```
@@ -0,0 +1,664 @@
+---
+id: ocsf
+name: OCSF
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+package_icon: https://github.com/ocsf.png
+description: |
+  A package that handles onboarding of OCSF mapping pipeline topics.
+  Based on OCSF version 1.3.0.
+  See also: https://schema.ocsf.io/1.3.0/
+
+pipelines:
+  # System Activity (1)
+  import-oscf-filesystem-activity:
+    name: Import OCSF File System Activity
+    description: Import OCSF File System Activity events that come from an event mapping pipeline.
+    disabled: false
+    definition: |
+      // tql2
+      subscribe "ocsf.file_system_activity"
+      import
+    restart-on-error: 1.0m
```

```suggestion
    restart-on-error: 1.0m
```

@dominiklohmann this crops up on every pipeline. Any idea how to factor it? Or should restarting on error perhaps be the default?

---

I think `restart-on-error: true` would be the better approach here to avoid hardcoding the default timeout.

---

I replaced all hardcodings with `true` in https://github.com/tenzir/library/pull/10/commits/f8b83409cae1b47a29a7c8d29145f43b2dbe311a

---

My preferred solution would be to not write anything by default.